Bibliotheque nationale de France.js

```diff
@@ -528,34 +528,29 @@ var BnfClass = function() {
 	*/
 	this.getSelectedItems = function(doc) {
 		var items = new Object();
-		
-		var baseUri = /^(https?:\/\/[^\/]+)/.exec(doc.location.href)[1];
-		var cellPath = '//td[@class="mn_partienoticesynthetique"]';
-		var spanPath = './/span';
+		var cellPath = '//div[@class="liste-notices"]/div[@class="notice-item"]';
 		var cells = doc.evaluate(cellPath, doc, null, XPathResult.ANY_TYPE, null);
-		var cell = undefined;
-		var regexLink = /\s*window.location='([^']+)'\s*/;
-		
-		//Cell loop
 		while(cell = cells.iterateNext()) {
```

Comment:
declare `cell` as function-local (`var cell;`). Right now it's in the global scope.